travis/build-all-in-one-image.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Only build docker container for master branch
+if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo 'upload to Docker Hub'; else echo 'skip docker upload for PR'; exit 0; fi
+
+export REPO=jaegertracing/all-in-one
+export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
+export TAG=`if [ "$BRANCH" == "master" ]; then echo "latest"; else echo "${BRANCH///}"; fi`
+echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, REPO=$REPO, PR=$PR, BRANCH=$BRANCH, TAG=$TAG"
+
+make build-all-in-one-linux || { echo 'failed build-all-in-one-linux' ; exit 0; }
```

Comment:
do `set -ex` at the top of the script instead, and it will fail on errors automatically